gsa/public/locales/gsa-de.json

```diff
@@ -178,6 +181,7 @@
   "Base Score": "Basisscore",
   "Base URL": "Basis URL",
   "Base Vector": "Basisvektor",
+  "Base with a minimum set of NVTs": "Basis Konfiguration mit einer minimalen Menge an NVTs",
```

Comment:
Maybe better
"Basis Konfiguration mit einer minimalen Anzahl an NVTs" or
"Basis Konfiguration mit einem minimalen Satz von NVTs"

Reply:
As above, this translation was already in GSA, I just put it back in. I can change it, of course.

Reply:
You can decide what's best.